src/mono/mono/utils/options-def.h

```diff
@@ -142,6 +142,12 @@ DEFINE_INT(jiterpreter_interp_entry_queue_flush_threshold, "jiterpreter-interp-e
 DEFINE_INT(jiterpreter_wasm_bytes_limit, "jiterpreter-wasm-bytes-limit", 6 * 1024 * 1024, "Disable jiterpreter code generation once this many bytes of WASM have been generated")
 #endif // HOST_BROWSER
 
+#ifdef HOST_WASM
```

Comment:
Should it include `HOST_IOS` as well?

Reply:
Its better to start with one platform to see how it goes.